.github/syncRepoFiles/index.js

```diff
@@ -288,7 +288,7 @@ class Git {
                 return branch;
             }
             catch (error) {
-                core.error(error);
+                core.info(`${error}`);
```

Comment:
Does output of this error include call stack and error details?

Reply:
only show "HttpError: Branch not found". No call stack ~